sections/deckbuilding.tex

```diff
@@ -173,54 +171,102 @@ \subsection*{\pagetarget{spells}{Spell Cards}}
     \draw (0.8, -2.8) node {\encircle{5}};
   \end{tikzpicture}\\
   \medskip
-  \footnotesize{\textbf{\textit{\textcolor{darkcandyapplered}{Spell Card}}}}
-  \scriptsize
-  \begin{multicols}{2}
-    \begin{itemize}[itemsep=5pt]
-      \item[\textbf{1.}] \textbf{Spell Name}
-      \item[\textbf{2.}] \textbf{School of Magic}
-      \item[\textbf{3.}] \textbf{Cost\\to Empower}
-      \item[\textbf{4.}] \textbf{Spell Effect}
-      \item[\textbf{5.}] \textbf{Alternative\\Effect}
-      \item[]
+  \footnotesize
+  \textbf{\textit{\textcolor{darkcandyapplered}{Spell Card}}}
+
+  \noindent
+  \begin{minipage}[t]{0.5\linewidth}
+    \begin{itemize}[itemsep=0pt, parsep=5pt, topsep=0pt, partopsep=0pt]
+      \item[\textbf{1.}] Spell Name
+      \item[\textbf{2.}] School of Magic
+      \item[\textbf{3.}] Cost to Empower
     \end{itemize}
-  \end{multicols}
+  \end{minipage}
+  \begin{minipage}[t]{0.4\linewidth}
+    \begin{itemize}[itemsep=0pt, parsep=5pt, topsep=0pt, partopsep=0pt]
+      \item[\textbf{4.}] Spell Effect
+      \item[\textbf{5.}] Alternative\\Effect
+    \end{itemize}
+  \end{minipage}
 }
+\medskip
 
 Spells can be gained by Building the Mage Guild\index{Mage Guild}.
 Building it also enables you to use the Spell Book Token\index{Spell Book Token} for buying additional Spells.
 The Token cannot be used during the \textbf{same Round} when the Guild is constructed. Spells can also be \pagelink{Trading}{traded} in Alliance and Cooperative Scenarios.
 
 During Combat, \textbf{only one Spell Card} may be played by each player \textbf{per Combat Round}.
 
-\note{6}{
-  When you use an effect which lets you cast a Spell from somewhere other than your hand (such as directly from the Spell Deck), place that Spell into your Discard Pile after casting it.
-}
+\columnbreak
+When you use an effect which lets you cast a Spell from somewhere other than your hand (such as directly from the Spell Deck), place that Spell into your Discard Pile after casting it.
+
+\begin{center}
+  \vspace*{\fill}
+  \includegraphics[width=0.9\linewidth]{\art/air_magic.png}
```

Comment:
I'm introducing transparency package in the other PR. I think it's better to have the art non-transparent, and regulate the alpha channel with LaTeX moving forward.

Reply:
I agree